packages/palette_generator/lib/palette_generator.dart

```diff
@@ -96,12 +95,11 @@ class PaletteGenerator with Diagnosticable {
   /// The [image] must not be null.
   static Future<PaletteGenerator> fromImage(
     ui.Image image, {
-    Rect region,
-    int maximumColorCount,
-    List<PaletteFilter> filters,
-    List<PaletteTarget> targets,
+    Rect? region,
+    int? maximumColorCount,
+    List<PaletteFilter>? filters,
+    List<PaletteTarget>? targets,
```

Comment:
It's worth considering whether these should really be nullable, vs. having empty set default values.

Reply:
https://github.com/flutter/packages/pull/287#discussion_r576958171